### ISSUE TITLE
ERROR: Function Clause Error on Care Team View (for SU)

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/healthlocker#934]
Logged in as tkestedt3tkest@outlook.com going to ‘Care Team’ on the menu gives an error screen:

![screenshot 2017-07-18 12 48 44](https://user-images.githubusercontent.com/24604903/28316439-cf8bd47a-6bba-11e7-8daf-2d3ff9c3da41.png)

Tried a different user and got the same problem:
user: tkest152tkest@outlook.com
![screenshot 2017-07-18 13 13 15](https://user-images.githubusercontent.com/24604903/28316464-eb0b5860-6bba-11e7-8876-1c97e8892629.png)

**Comment by katyedwards** [comment]
? relates to the issue I closed this morning #918?

**Comment by katyedwards** [comment]
Garry has identified _another_ error, but I think it is the same as this issue and not something _new_...

Description:
The error below is generated by HealthLocker when there is a change to the team member caseload i.e. the clinician involvement is ended.
Team Members dropping out of the care team would be a common procedure.
 
Steps:
 
ð  Ensure ‘Connect with my care team and health record’ is place and that  Care Team function is working
ð  Upon ePJS, make a change to Team Member associated with HealthLocker e.g. end team member involvement. 
ð  Update MHL2 extract to reflect the change.
ð  Using either the Service User or Carer Healthlocker account – login
ð  Click Care Team’

![image002](https://user-images.githubusercontent.com/24604903/28512318-4e608254-7049-11e7-874d-19928f6c2eae.jpg)

**Comment by katyedwards** [comment]
Note: Apparently same problem exists for both SUs and Carers if a chance is made to the care team assigned.

Oddly, the disconnecting/ reconnecting from SLaM account apparently works to 'fix' this for SUs but not for carers.

**Comment by katyedwards** [comment]
Any chance #853 has anything to do with this?

**Comment by katbow** [comment]
Forgot to update this earlier - since the error always appears to be when *adding* clinicians to the room this was the first section we investigated. We believe the error is due to the fact that duplicate clinicians were being inserted into this table (since they were not removed before). 

We need to ensure the clinicians are not added in duplicate, either by deleting all previous clinicians in the room (this will not affect room history, it will all still exist and be there when the clinician visits the room), or not attempting to insert clinicians which have already been added to the room.
